test/e2e/shoot/internal/logging/event_logging.go

```diff
@@ -0,0 +1,264 @@
+	kubeSystemNamespace    = "kube-system"
+	operatorID             = "operator"
+	lokiName               = "loki"
+	gardenNamesapce        = "garden"
+	fluentBitDaemonSetName = "fluent-bit"
+
+	defaultPollInterval = 5 * time.Second
+	lokiContainer       = "loki"
+	lokiPort            = 3100
+)
+
+var (
+	lokiLabels = map[string]string{
+		"app":  "loki",
+		"role": "logging",
+	}
+)
+
+// EventLoggingVerifier verifies the event-logger.
+type EventLoggingVerifier struct {
+	*framework.ShootCreationFramework
+	shootNamespace string
+	seedEvent      *corev1.Event
+	shootEvent     *corev1.Event
+}
+
+// Before is called before the test is started and checks for required logging components.
+func (v *EventLoggingVerifier) Before(ctx context.Context) {
+	v.shootNamespace = v.Shoot.Status.TechnicalID
+	v.seedEvent = getEventFor("seed", v.shootNamespace)
+	v.shootEvent = getEventFor("shoot", kubeSystemNamespace)
+
+	Eventually(func(g Gomega) {
+		g.Expect(v.ShootFramework.SeedClient.Client().Get(ctx, client.ObjectKey{Namespace: gardenNamesapce, Name: fluentBitDaemonSetName}, &appsv1.DaemonSet{})).To(Succeed())
+		g.Expect(v.ShootFramework.SeedClient.Client().Get(ctx, client.ObjectKey{Namespace: v.shootNamespace, Name: lokiName}, &appsv1.StatefulSet{})).To(Succeed())
+		g.Expect(v.ShootFramework.SeedClient.Client().Get(ctx, client.ObjectKey{Namespace: v.shootNamespace, Name: eventLogger}, &appsv1.Deployment{})).To(Succeed())
+	}).Should(Succeed())
+
+	By("Waite for the shoot Loki and Event-Logger to become ready")
+	Expect(v.WaitUntilStatefulSetIsRunning(ctx, lokiName, v.shootNamespace, v.ShootFramework.SeedClient)).To(Succeed())
+	Expect(v.WaitUntilDeploymentIsReady(ctx, eventLogger, v.shootNamespace, v.ShootFramework.SeedClient)).To(Succeed())
+}
+
+// Prepare is called after Before and it makes all preparation for the test.
+func (v *EventLoggingVerifier) Prepare(ctx context.Context) {
+	By("Create seed and shoot events")
+	Eventually(func(g Gomega) {
+		g.Expect(v.ShootFramework.SeedClient.Client().Create(ctx, v.seedEvent)).To(Succeed())
+		g.Expect(v.ShootFramework.ShootClient.Client().Create(ctx, v.shootEvent)).To(Succeed())
+	}).Should(Succeed())
+}
+
+// Expect is the the process were we expect to get the correct result from the logging test.
+func (v *EventLoggingVerifier) Expect(ctx context.Context) {
+	By("Wait until Loki receive seed event")
+	Expect(waitUntilLokiReceivesEvent(ctx, v.Logger, 30*time.Second, lokiLabels, operatorID, v.shootNamespace, `origin_extracted="seed",source="event-logger",sourceHost="seed"`, []string{eventMessageForSeed}, time.Now().Add(-5*time.Minute), v.ShootFramework.SeedClient)).To(Succeed())
+
+	By("Wait until Loki receive shoot event")
+	Expect(waitUntilLokiReceivesEvent(ctx, v.Logger, 30*time.Second, lokiLabels, operatorID, v.shootNamespace, `origin_extracted="shoot",source="event-logger",sourceHost="shoot"`, []string{eventMessageForShoot}, time.Now().Add(-5*time.Minute), v.ShootFramework.SeedClient)).To(Succeed())
```

Comment:
There are many parameters that always have the same value. Please drop those parameters.

Also, instead of passing `time.Now().Add(-5*time.Minute)`, should we pass the events' `creationTimestamp`?